tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/i/Rule_I_46_IncorrectImagePath.java

```diff
@@ -0,0 +1,165 @@
+/*******************************************************************************
+ * Copyright (c) 2023 THALES GLOBAL SERVICES.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.test.validation.rules.ju.testcases.i;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IMarker;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.emf.common.util.Diagnostic;
+import org.eclipse.emf.ecore.EClass;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.validation.preferences.EMFModelValidationPreferences;
+import org.eclipse.sirius.business.api.query.DViewQuery;
+import org.eclipse.sirius.business.api.session.Session;
+import org.eclipse.sirius.diagram.DNode;
+import org.eclipse.sirius.diagram.WorkspaceImage;
+import org.eclipse.sirius.diagram.ui.business.api.image.WorkspaceImageHelper;
+import org.polarsys.capella.common.data.modellingcore.ModellingcorePackage;
+import org.polarsys.capella.common.helpers.EcoreUtil2;
+import org.polarsys.capella.common.tools.report.appenders.reportlogview.LightMarkerRegistry;
+import org.polarsys.capella.common.tools.report.appenders.reportlogview.MarkerViewHelper;
+import org.polarsys.capella.core.data.capellacore.CapellaElement;
+import org.polarsys.capella.core.libraries.model.CapellaModel;
+import org.polarsys.capella.core.libraries.utils.ScopeModelWrapper;
+import org.polarsys.capella.core.model.handler.markers.ICapellaValidationConstants;
+import org.polarsys.capella.core.model.handler.provider.CapellaAdapterFactoryProvider;
+import org.polarsys.capella.core.model.handler.validation.CapellaDiagnostician;
+import org.polarsys.capella.core.platform.sirius.sirius.validation.ddiagram.ImagePathRemoveResolver;
+import org.polarsys.capella.shared.id.handler.IScope;
+import org.polarsys.capella.shared.id.handler.IdManager;
+import org.polarsys.capella.test.framework.api.OracleDefinition;
+import org.polarsys.capella.test.validation.rules.ju.testcases.ValidationRulePartialTestCase;
+
+/**
+ * Test for I_46 quick fix : remove not found image on element style in diagram
+ * @author Séraphin Costa
+ *
+ */
+public class Rule_I_46_IncorrectImagePath extends ValidationRulePartialTestCase {
+
+    private static final String INVALID_DNODE = "_ecDJsCgEEeuc881X317kNw";
```

Comment:
you could have updated `exchange-item-instance-and-part-model.aird` and `exchange-item-instance-and-part-model.capella` to add a OperationalCapability and an operational Entity and a [OCB] diagram to test both node with default image and other nodes